bids-validator/bids_validator/rules/file_level_rules.json

```
@@ -526,7 +532,7 @@
   },
 
   "microscopy": {
-    "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?microscopy[\\/\\\\](sub-[a-zA-Z0-9]+)(?:(_ses-[a-zA-Z0-9]+))?(?:_sample-[a-zA-Z0-9]+)(?:_acq-[a-zA-Z0-9]+)?(?:_stain-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:_chunk-[a-zA-Z0-9]+)?(((@@@_microscopy_type_@@@)(@@@_microscopy_ext_@@@))|(_photo\\.jpg))$",
+    "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?microscopy[\\/\\\\](sub-[a-zA-Z0-9]+)(?:(_ses-[a-zA-Z0-9]+))?(?:_sample-[a-zA-Z0-9]+)(?:_acq-[a-zA-Z0-9]+)?(?:_stain-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:_chunk-[a-zA-Z0-9]+)?(((@@@_microscopy_type_@@@)(@@@_microscopy_ext_@@@))(@@@_photo_ext_@@@))$",
```

```suggestion
    "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?micr[\\/\\\\](sub-[a-zA-Z0-9]+)(?:(_ses-[a-zA-Z0-9]+))?(?:_sample-[a-zA-Z0-9]+)(?:_acq-[a-zA-Z0-9]+)?(?:_stain-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:_chunk-[a-zA-Z0-9]+)?(((@@@_microscopy_type_@@@)(@@@_microscopy_ext_@@@))(@@@_photo_ext_@@@))$",
```

You may want to change `microscopy` to `micr` in this PR as well, as you prefer.

---

Done